package.json

```diff
@@ -38,7 +38,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@babel/plugin-proposal-private-methods": "^7.18.6",
-    "@compodoc/compodoc": "^1.1.9",
+    "@compodoc/compodoc": "1.1.21",
```

Comment:
Pinning compodoc because the downstream dep `@angular-devkit/schematics` appears to have dropped < Node 16 in the latest version which was breaking CI for us.